o-fish-ios/Views/ReportFlow/Crew/CrewView.swift

```diff
@@ -80,7 +80,7 @@ struct CrewView: View {
                 }
                 Spacer()
             }
-            .showingAlert(alertItem: $showingAlertItem)
+            .showingAlert(alertItem: self.$showingAlertItem)
```

Comment:
Why use `self`? Our coding guidelines are only to use it when required by the compliler

Reply:
If we don't use self in this place our CI is failed.